addon/session-stores/adaptive.js

```diff
@@ -133,6 +133,11 @@ export default Base.extend({
       const options = this.getProperties('cookieDomain', 'cookieName', 'cookieExpirationTime', 'cookiePath');
       options._fastboot = this.get('_fastboot');
       options._cookies = this.get('_cookies');
+
+      let expirationCookieName = `${this.get('cookieName')}-expiration_time`;
+      let expirationTime = parseInt(options._cookies.read(expirationCookieName), 10);
+      this.set('cookieExpirationTime', expirationTime);
```

Comment:
I wasn't sure whether to add this, or change the test at https://github.com/simplabs/ember-simple-auth/pull/1257/files#diff-37b831f2713acdc8f0acf654aa735d70R310.

If the store is an `Adaptive` store, should we do something like
```
expect(store.get('_cookies.cookieExpirationTime')).to.equal(expirationTime);
```
?